boxoffice/static/js/models/util.js

```diff
@@ -21,23 +21,39 @@ export const Util = {
   }
 }
 
-export const fetch = function(config){
+export const fetch = function (config){
   return $.ajax({
     url: config.url,
     dataType: 'json'
   });
 }
 
-export const post = function(config){
+export const post = function (config){
   return $.ajax({
     url: config.url,
     type: 'POST',
     dataType: 'json'
   });
 }
 
-export const scrollToElement = function(element, speed=500) {
+export const scrollToElement = function (element, speed=500) {
   $('html,body').animate({
     scrollTop: $(element).offset().top
   }, speed);
 }
+
+export const makePageTitle = function (title, subTitle) {
+  /* Takes title, subTitle and join them into a string using " — ".
```

Comment:
"Takes a `title` and a `sub-title` and returns a concatenated string separated by " - "".